docs/overview.asciidoc

```diff
@@ -1,9 +1,15 @@
 [[overview]]
 == Overview
 
-This is the official PHP client for Elasticsearch.  It is designed to be a very low-level client that does not stray from the REST API.
-
-All methods closely match the REST API, and furthermore, match the method structure of other language clients (ruby, python, etc).  We hope that this consistency makes it easy to get started with a client, and to seamlessly switch from one language to the next with minimal effort.
-
-The client is designed to be "unopinionated".  There are a few universal niceties added to the client (cluster state sniffing, round-robin requests, etc) but largely it is very barebones.  This was intentional.  We want a common base that more sophisticated libraries can build on top of.
-
+This is the official PHP client for {es}. It is designed to be a low-level 
```

Comment:
Just a curiosity, {es} is a placeholder?

Reply:
Yes. It is an attribute from the attributes list that available in the books if their `index.asciidoc` file contains https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc. I checked it and the PHP book doesn't use the attributes, so I change {es} back to Elasticsearch. Thanks for the heads-up!

https://github.com/elastic/docs/blob/c2b2e5be8711f5a8bb277a79916814f922817281/shared/attributes.asciidoc#L97

Reply:
I just approved #1000, you can merge this PR. Thanks!